chain/jsonrpc/src/lib.rs

```diff
@@ -236,6 +251,138 @@ impl JsonRpcHandler {
+                    )
+                }
+                "adv_accept_connection_from" => {
+                    let req =
+                        near_jsonrpc_primitives::types::requests::AcceptConnectionFromRequest::parse(
+                            params,
+                        )?;
+                    let address = req.address.parse().map_err(|_| {
+                        RpcError::new_internal_error(
+                            None,
+                            format!("Failed to parse address: {:?}", req.address),
+                        )
+                    })?;
+                    let result = self
+                        .peer_manager_addr
+                        .clone()
+                        .send(AcceptConnectionFrom { addr: address })
+                        .await?;
+                    Some(
+                        serde_json::to_value(result)
+                            .map_err(|err| RpcError::serialization_error(err.to_string())),
+                    )
+                }
+                "adv_get_peer_id" => {
+                    let response = self.peer_manager_addr.send(GetPeerId {}).await?;
+                    Some(
+                        serde_json::to_value(response)
+                            .map_err(|err| RpcError::serialization_error(err.to_string())),
+                    )
+                }
+                "adv_get_peer_list" => {
+                    let result = self.peer_manager_addr.send(PeersRequest {}).await?;
+                    Some(
+                        serde_json::to_value(result)
+                            .map_err(|err| RpcError::serialization_error(err.to_string())),
+                    )
+                }
+                "adv_get_network_stats" => {
+                    let result = self.peer_manager_addr.send(GetNetworkStats {}).await?;
+                    Some(
+                        serde_json::to_value(result)
+                            .map_err(|err| RpcError::serialization_error(err.to_string())),
+                    )
+                }
+                "adv_set_routing_table" => {
+                    let request = SetRoutingTableRequest::parse(params)?;
+                    let result = self
+                        .peer_manager_addr
+                        .send(SetRoutingTableRequest { edges_info: request.edges_info })
+                        .await?;
+                    Some(
+                        serde_json::to_value(result)
+                            .map_err(|err| RpcError::serialization_error(err.to_string())),
+                    )
+                }
+                "adv_network_info" => {
+                    let network_info_response = self.network_info().await?;
+                    Some(
+                        serde_json::to_value(network_info_response)
+                            .map_err(|err| RpcError::serialization_error(err.to_string())),
+                    )
+                }
```

Comment:
We already have a rpc endpoint for network info. Why do we duplicate it here?

Reply:
I'll remove it, this is no longer needed I'll do this in a different way.
My plan is to have a query rpc, where you specify what you want to ask for.

This will allow me to specify what I want to get from peer based on query arguments, for example
`{
     routing_table: Optional<...>,
     connected_peer_list: Optional<...>
    ... etc
}
`   
This will allow me to create custom queries, requesting only fields I need.